hack/base-images/README.md

```diff
@@ -0,0 +1,29 @@
+# Create docker image from Centos ISO
```

Comment:
This is a one time process to create minimal image from ISO. not sure if it belongs in this repo.

Reply:
I think its fine to keep here.
We will want to create additional images with new versions.

Reply:
sounds good. I will add automating this script in teamcity in next couple of sprint then.